tests/functional/rbac/test_reassignment.py

```diff
@@ -149,6 +154,137 @@ def grant_role(self, client: ADCMClient, user: User, role: RbacRoles, *objects:
                 name=f'{user.username} is {role.value}', role=client.role(name=role.value), objects=objects, user=[user]
             )
 
+    def test_add_remove_user_from_group_and_policy(self, clients, is_denied_to_user, prepare_objects, user):
+        """Test that policies are applied after user add/remove to the policy"""
+
+        admin_cluster, admin_service, admin_component, *_ = prepare_objects
+
+        is_denied_to_user(admin_cluster, BusinessRoles.EditClusterConfigurations)
+        is_denied_to_user(admin_service, BusinessRoles.EditServiceConfigurations)
+        is_denied_to_user(admin_component, BusinessRoles.EditComponentConfigurations)
+        with allure.step('Create user group'):
+            test_group = clients.admin.group_create("Test_group", user=[{"id": user.id}])
+        with allure.step('Create a test policy with user group'):
```

Comment:
I suggest 'Create a "Cluster Administrator" policy for a group'. That way we clarify what we grant to which objects (users/groups)

Reply:
it is totally not significant what policy do we use in this case, but okay, I'll change, if you think it is